### PR TITLE
minor fix in example

### PR DESCRIPTION
**Pull request by tareqalayan**

### CODE REVIEW

File: examples/inventory/install_engine.inv [hunk 1]
```diff
@@ -74,7 +74,7 @@ database.example.com
 
 [database:vars]
 # Password for SSH connection to database host
-ansible_ssh_pass=qum5net
+ansible_ssh_pass=fillme
```
Comment (lukas-bednar) [inline examples/inventory/install_engine.inv]: Honestly I would remove this line, the user might use ssh-key, and doesn't care about password.
I think it doesn't belong here.

Comment (StLuke) [inline examples/inventory/install_engine.inv]: +1

Comment (lukas-bednar) [inline examples/inventory/install_engine.inv]: There is also one occurrence at line 4.

Comment (StLuke) [inline examples/inventory/install_engine.inv]: and also other inventory files

**Comment by lukas-bednar** [comment]
Fixes #117

**Comment by StLuke** [comment]
so why not remove it in all the inventory files in one commit, as it was pointed out in the comment on the patch?

**Comment by tareqalayan** [comment]
didn't look in other files ...  sorry 

i will remove them soon ..

**Comment by StLuke** [comment]
https://github.com/rhevm-qe-automation/ovirt-ansible/issues/119

**Pull request merged**